app/views/guide_form_elements.html

```diff
@@ -145,6 +145,23 @@ <h3 class="heading-medium" id="form-file-upload">File upload</h3>
 </code>
 </pre>
 
+  <div class="text">
+    <p>
+      We recommend using a native input using <code class="code">type="file"</code>, rather than a custom implementation.
+    </p>
+    <p class="lead-in">
+      This is so:
+    </p>
+    <ul class="list list-bullet">
+      <li>the control gains focus when tabbing through the page</li>
+      <li>the control functions using a keyboard</li>
+      <li>the control functions using assistive technology</li>
+      <li>the control functions when javascript is disabled</li>
```

Comment:
should we generally refer to 'javascript not available', given [the majority case is not 'disabled'](https://gds.blog.gov.uk/2013/10/21/how-many-people-are-missing-out-on-javascript-enhancement/)